wp-content/themes/goonj-crm/functions.php

```diff
@@ -269,7 +276,7 @@ function goonj_handle_user_identification_form() {
 		->execute();
 
 		if ($purpose === 'dropping-center') {
-			wp_redirect(get_home_url() . "/dropping-center/#?Collection_Camp_Core_Details.Contact_Id=" . $found_contacts['id']);
+			wp_redirect(get_home_url() . "/dropping-center/location/#?Collection_Camp_Core_Details.Contact_Id=" . $found_contacts['id']);
```

Comment:
Why have you named it location?

Reply:
In dropping center landing page after email & phone details submitted existing user is redirected to fill location details for dropping center